zerver/lib/test_classes.py

```diff
@@ -1480,13 +1487,28 @@ def check_webhook(
             headers = get_fixture_http_headers(self.WEBHOOK_DIR_NAME, fixture_name)
             headers = standardize_headers(headers)
             kwargs.update(headers)
+        try:
+            msg = self.send_webhook_payload(
+                self.test_user,
+                self.url,
+                payload,
+                **kwargs,
+            )
+        except EmptyResponseError:
+            if except_noop:
+                return
+            else:
+                raise  # nocoverage
```

Comment:
I'm going to switch this to `raise AssertionError`, which has the same effect for coverage but lets us provide a string advertising `expect_noop`.